docs/src/Home.stories.mdx

```diff
@@ -25,6 +25,6 @@ View their code in the `.stories.tsx` files to learn how they work.
   </li>
 </ul>
 
-
 ## Usage Issues
- * Pseudo states will not render correctly when switching between stories in Docs mode. Toggling between Docs/Canvas mode will reset the pseudo states.
+
+- Pseudo states will not render correctly when switching between stories in Docs mode. Toggling between Docs/Canvas mode will reset the pseudo states.
```

Comment:
I didn't do any changes here. When I save this file, `*` was changed to `-` automatically. Anyone who knows why can share with me, I appreciate it.